seed/views/01002_region_caseload_view.js

```diff
@@ -0,0 +1,28 @@
+exports.seed = function (knex, Promise) {
+  var sql = `CREATE VIEW app.region_caseload_view
+        WITH SCHEMABINDING
+        AS
+        SELECT     
+          MAX(r.id) AS id
+          , l.id AS link_id
+          , MAX(l.description) AS name
+          , grade_code
+          , SUM(untiered) AS untiered
+          , SUM(d2) AS d2
+          , SUM(d1) AS d1
+          , SUM(c2) AS c2
+          , SUM(c1) AS c1
+          , SUM(b2) AS b2
+          , SUM(b1) AS b1
+          , SUM(a) AS a
+          , SUM(total_cases) AS total_cases
+        FROM app.ldu_caseload_view lv  
+          JOIN app.ldu l ON lv.id = l.id
+          JOIN app.region r ON l.region_id = r.id
```

Comment:
Is this join to region needed? Can we just use `l.region_id`? Is there a foreign key constraint to ensure that `l.region_id` is valid?

Reply:
Also in the team view

Reply:
removed